conf/configs/dev.yml

```diff
@@ -34,3 +34,9 @@ aco_config:
       data:
         - "adjudicated"
         - "partially-adjudicated"
+    - model: "TCOCMD"
```

Comment:
Do we want to call this TCOCMD or just something like TESTIGNORE?  Same goes for test and opensbx env's.  Thinking that if we put up example code on our static site that referencing a specific company might not be ideal.

Reply:
Yeah, that's true. Because this is technically a new APM, I wanted to follow the format of the config file; the shorthand name of the model with the regex for the possible ID it may have. We can chat more about this!